bytecomp/bytelink.ml

```diff
@@ -525,13 +525,13 @@ let link_bytecode_as_c tolink outfile with_main =
          (Marshal.to_string (Symtable.initial_global_table()) []);
        output_string outchan "\n};\n\n";
        (* The sections *)
-       let sections : (string * Obj.t) list =
-         [ Bytesections.Name.to_string SYMB,
-           Symtable.data_global_map();
-           Bytesections.Name.to_string PRIM,
-           Obj.repr(Symtable.data_primitive_names());
-           Bytesections.Name.to_string CRCS,
-           Obj.repr(extract_crc_interfaces()) ]
+       let sections : (string * Obj.t) array =
+         [| Bytesections.Name.to_string SYMB,
+            Symtable.data_global_map();
+            Bytesections.Name.to_string PRIM,
+            Obj.repr(Symtable.data_primitive_names());
```

Comment:
I think that with this new scheme, there's no longer a need for `PRIM` section in `-custom` mode. When linking a `-custom` executable, the available primitives are recorded in the camlprim file for the runtime's benefit (via `caml_build_primitive_table_builtin`) and then _again_ in the `PRIM` section so that `Meta` can read it.